util/cmake/FindSeqAn.cmake

```diff
@@ -174,6 +171,8 @@ endif ()
 # Require C++14
 # ----------------------------------------------------------------------------
 
+# The standard library of visual studio defines __cplusplus still as
```

Comment:
the macro is not defined by the standard library, but by the compiler. Clang/c2 correctly sets the macro!